test/message_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/morse_translator'
+require './lib/message'
+
+class MessageTest < Minitest::Test
+
+  def test_it_exists
+    morse_translator = MorseTranslator.new
+    message = Message.new("There are 3 ships", morse_translator)
+    assert_instance_of Message, message
+  end
+
+  def test_it_has_text
+    morse_translator = MorseTranslator.new
+    message = Message.new("There are 3 ships", morse_translator)
+    assert_equal "There are 3 ships", message.text
```

Comment:
This is interesting. I know that my recommendation above would mean that this test didn't pass. I think for now it's probably okay as is. I like the fact that you're being diligent in your testing, and probably prefer having this test over not having the instance variable. Good call!